recipe/migrations/arch_rebuild.txt

```diff
@@ -36,6 +36,7 @@ bayes-filters-lib
 bayeswave
 bazel
 bcrypt
+beagle-lib
```

Comment:
`beagle-lib` is a Bioconda recipe. There is no feedstock for it at conda-forge.

Reply:
This branch has been merged to master, do i need to delete the beagle-lib ?

Reply:
Yes, please open a new PR to remove it!

Reply:
ok, thank you.